lib/ohmqtt/ohmqtt.cpp

```diff
@@ -8,31 +8,42 @@
 ohmqtt::ohmqtt() {
   IPAddress broker(10,4,2,21);    // Address of the MQTT broker
```

Comment:
Ideally I should be passing that in via a build flag. That was a future me problem to sort out.